seleniumByTechnocredits/src/seleniumPractise/prachiJ/Assingment1/AlertDemo_Q1.java

```diff
@@ -0,0 +1,53 @@
+package seleniumPractise.prachiJ.Assingment1;
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import util.Init;
+
+/*
+ * Alert Demo > Submit > Alert
+ * Validation of the values of Alert Demo with of the Alert.
+ */
+public class AlertDemo_Q1 {
+	public static void main(String[] args) throws InterruptedException {
+		WebDriver driver = Init.initChromeDriver();
+		String url = "http://automationbykrishna.com/#";
+		driver.get(url);
+		Thread.sleep(2000);
+		driver.findElement(By.xpath("//*[@class='nav navbar-nav navbar-right']/preceding-sibling::ul/li[4]/a")).click();
```

Comment:
Could you please write exact xpath (tag name) instead of//* as we have seen that if we write * then it goes to each element in that DOM to check the values.

Reply:
Ammended suggested changes